apps/api-extractor-model/src/items/ApiItem.ts

```diff
@@ -64,15 +65,16 @@ export const ApiItem_parent: unique symbol = Symbol('ApiItem._parent');
 export class ApiItem {
   public [ApiItem_parent]: ApiItem | undefined;
 
-  public static deserialize(jsonObject: IApiItemJson): ApiItem {
+  public static deserialize(jsonObject: IApiItemJson, context: DeserializerContext): ApiItem {
```

Comment:
I am curious why everywhere else `context` comes before `jsonObject`, except for here? Everything looks mapped correctly, I'm just wondering what the reasoning is. Thanks.

Reply:
Everywhere else the first parameter is the output (`IBlahOptions`). Here the first parameter is the input (`IBlahJson`).